[2025-05-03T08:04:49.958+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-03T08:04:51.091+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: diabetes_data_pipeline.save_validation_data manual__2025-05-03T08:03:23.005332+00:00 [queued]>
[2025-05-03T08:04:51.591+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: diabetes_data_pipeline.save_validation_data manual__2025-05-03T08:03:23.005332+00:00 [queued]>
[2025-05-03T08:04:51.592+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-05-03T08:04:51.979+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): save_validation_data> on 2025-05-03 08:03:23.005332+00:00
[2025-05-03T08:04:52.339+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'diabetes_data_pipeline', 'save_validation_data', 'manual__2025-05-03T08:03:23.005332+00:00', '--job-id', '256', '--raw', '--subdir', 'DAGS_FOLDER/process_and_clean_data.py', '--cfg-path', '/tmp/tmp6mns98dy']
[2025-05-03T08:04:52.350+0000] {standard_task_runner.py:91} INFO - Job 256: Subtask save_validation_data
[2025-05-03T08:04:52.319+0000] {standard_task_runner.py:63} INFO - Started process 2134 to run task
[2025-05-03T08:04:53.320+0000] {task_command.py:426} INFO - Running <TaskInstance: diabetes_data_pipeline.save_validation_data manual__2025-05-03T08:03:23.005332+00:00 [running]> on host e89618938543
[2025-05-03T08:04:54.545+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='diabetes_data_pipeline' AIRFLOW_CTX_TASK_ID='save_validation_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-03T08:03:23.005332+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-03T08:03:23.005332+00:00'
[2025-05-03T08:04:54.569+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-03T08:05:11.071+0000] {process_and_clean_data.py:561} INFO - Guardando datos procesados en tabla diabetes_validation_processed...
[2025-05-03T08:05:12.437+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-05-03T08:05:12.463+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:12.568+0000] {sql.py:457} INFO - Running statement: DROP TABLE IF EXISTS diabetes_validation_processed, parameters: None
[2025-05-03T08:05:13.051+0000] {sql.py:466} INFO - Rows affected: 0
[2025-05-03T08:05:13.232+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:13.407+0000] {sql.py:457} INFO - Running statement: 
        CREATE TABLE diabetes_validation_processed (
            `feature_0` FLOAT, `feature_1` FLOAT, `feature_2` FLOAT, `feature_3` FLOAT, `feature_4` FLOAT, `feature_5` FLOAT, `feature_6` FLOAT, `feature_7` FLOAT, `feature_8` FLOAT, `feature_9` INT, `feature_10` INT, `feature_11` INT, `feature_12` INT, `feature_13` INT, `feature_14` INT, `feature_15` INT, `feature_16` INT, `feature_17` INT, `feature_18` INT, `feature_19` INT, `feature_20` INT, `feature_21` INT, `feature_22` INT, `feature_23` INT, `feature_24` INT, `feature_25` INT, `feature_26` INT, `feature_27` INT, `feature_28` INT, `feature_29` INT, `feature_30` INT, `feature_31` INT, `feature_32` INT, `feature_33` INT, `feature_34` INT, `feature_35` INT, `feature_36` INT, `feature_37` INT, `feature_38` INT, `feature_39` INT, `feature_40` INT, `feature_41` INT, `feature_42` INT, `feature_43` INT, `feature_44` INT, `feature_45` INT, `feature_46` INT, `feature_47` INT, `feature_48` INT, `feature_49` INT, `feature_50` INT, `feature_51` INT, `feature_52` INT, `feature_53` INT, `feature_54` INT, `feature_55` INT, `feature_56` INT, `feature_57` INT, `feature_58` INT, `feature_59` INT, `feature_60` INT, `feature_61` INT, `feature_62` INT, `feature_63` INT, `feature_64` INT, `feature_65` INT, `feature_66` INT, `feature_67` INT, `feature_68` INT, `feature_69` INT, `feature_70` INT, `feature_71` INT, `feature_72` INT, `feature_73` INT, `feature_74` INT, `feature_75` INT, `feature_76` INT, `feature_77` INT, `feature_78` INT, `feature_79` INT, `feature_80` INT, `target` VARCHAR(255), `encounter_id` INT, `patient_nbr` INT
        )
        , parameters: None
[2025-05-03T08:05:13.993+0000] {sql.py:466} INFO - Rows affected: 0
[2025-05-03T08:05:21.339+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:21.450+0000] {process_and_clean_data.py:693} ERROR - ✗ Error al guardar datos en tabla diabetes_validation_processed: (2002, "Can't connect to server on 'mysql_diabetes' (115)")
[2025-05-03T08:05:21.508+0000] {process_and_clean_data.py:697} INFO - Intentando método alternativo de guardado...
[2025-05-03T08:05:22.135+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:22.213+0000] {sql.py:457} INFO - Running statement: DROP TABLE IF EXISTS diabetes_validation_processed, parameters: None
[2025-05-03T08:05:22.528+0000] {sql.py:466} INFO - Rows affected: 0
[2025-05-03T08:05:22.538+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:22.643+0000] {sql.py:457} INFO - Running statement: 
            CREATE TABLE diabetes_validation_processed (
                `target` VARCHAR(255), `encounter_id` VARCHAR(50), `patient_nbr` VARCHAR(50), `feature_0` VARCHAR(50), `feature_1` VARCHAR(50), `feature_2` VARCHAR(50), `feature_3` VARCHAR(50), `feature_4` VARCHAR(50), `feature_5` VARCHAR(50), `feature_6` VARCHAR(50), `feature_7` VARCHAR(50), `feature_8` VARCHAR(50), `feature_9` VARCHAR(50), `feature_10` VARCHAR(50), `feature_11` VARCHAR(50), `feature_12` VARCHAR(50), `feature_13` VARCHAR(50), `feature_14` VARCHAR(50), `feature_15` VARCHAR(50), `feature_16` VARCHAR(50), `feature_17` VARCHAR(50), `feature_18` VARCHAR(50), `feature_19` VARCHAR(50)
            )
            , parameters: None
[2025-05-03T08:05:22.858+0000] {sql.py:466} INFO - Rows affected: 0
[2025-05-03T08:05:27.965+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:29.874+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:29.930+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 1: 100/20353
[2025-05-03T08:05:30.081+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:32.117+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:32.126+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 2: 200/20353
[2025-05-03T08:05:32.134+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:33.314+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:33.399+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 3: 300/20353
[2025-05-03T08:05:33.410+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:34.590+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:34.598+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 4: 400/20353
[2025-05-03T08:05:34.606+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:35.104+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:35.110+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 5: 500/20353
[2025-05-03T08:05:35.124+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:36.147+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:36.165+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 6: 600/20353
[2025-05-03T08:05:36.188+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:36.891+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:36.892+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 7: 700/20353
[2025-05-03T08:05:36.893+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:37.216+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:37.216+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 8: 800/20353
[2025-05-03T08:05:37.222+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:37.994+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:37.998+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 9: 900/20353
[2025-05-03T08:05:38.001+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:39.496+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:39.499+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 10: 1000/20353
[2025-05-03T08:05:39.500+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:40.536+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:40.539+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 11: 1100/20353
[2025-05-03T08:05:40.543+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:41.069+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:41.071+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 12: 1200/20353
[2025-05-03T08:05:41.072+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:41.351+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:41.352+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 13: 1300/20353
[2025-05-03T08:05:41.353+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:42.424+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:42.425+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 14: 1400/20353
[2025-05-03T08:05:42.426+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:42.839+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:42.840+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 15: 1500/20353
[2025-05-03T08:05:42.842+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:44.358+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:44.369+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 16: 1600/20353
[2025-05-03T08:05:44.371+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:45.740+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:45.771+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 17: 1700/20353
[2025-05-03T08:05:45.787+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:46.784+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:46.790+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 18: 1800/20353
[2025-05-03T08:05:46.810+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:47.207+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:47.219+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 19: 1900/20353
[2025-05-03T08:05:47.227+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:48.344+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:48.346+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 20: 2000/20353
[2025-05-03T08:05:48.347+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:48.953+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:48.954+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 21: 2100/20353
[2025-05-03T08:05:48.955+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:49.808+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:49.810+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 22: 2200/20353
[2025-05-03T08:05:49.816+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:50.946+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:50.955+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 23: 2300/20353
[2025-05-03T08:05:50.970+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:52.678+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:52.684+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 24: 2400/20353
[2025-05-03T08:05:52.731+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:53.354+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:53.354+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 25: 2500/20353
[2025-05-03T08:05:53.355+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:53.873+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:53.875+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 26: 2600/20353
[2025-05-03T08:05:53.880+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:55.057+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:55.072+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 27: 2700/20353
[2025-05-03T08:05:55.076+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:55.501+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:55.531+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 28: 2800/20353
[2025-05-03T08:05:55.715+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:56.569+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:56.569+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 29: 2900/20353
[2025-05-03T08:05:56.579+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:57.074+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:57.110+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 30: 3000/20353
[2025-05-03T08:05:57.121+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:57.584+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:57.587+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 31: 3100/20353
[2025-05-03T08:05:57.588+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:58.418+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:58.438+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 32: 3200/20353
[2025-05-03T08:05:58.453+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:59.300+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:59.307+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 33: 3300/20353
[2025-05-03T08:05:59.333+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:05:59.734+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:05:59.735+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 34: 3400/20353
[2025-05-03T08:05:59.737+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:00.885+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:00.932+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 35: 3500/20353
[2025-05-03T08:06:00.935+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:02.468+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:02.540+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 36: 3600/20353
[2025-05-03T08:06:02.557+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:03.378+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:03.379+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 37: 3700/20353
[2025-05-03T08:06:03.380+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:04.636+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:04.802+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 38: 3800/20353
[2025-05-03T08:06:04.854+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:05.198+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:05.204+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 39: 3900/20353
[2025-05-03T08:06:05.212+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:06.967+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:06.982+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 40: 4000/20353
[2025-05-03T08:06:06.987+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:07.514+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:07.516+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 41: 4100/20353
[2025-05-03T08:06:07.518+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:08.826+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:08.834+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 42: 4200/20353
[2025-05-03T08:06:08.835+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:10.552+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:10.681+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 43: 4300/20353
[2025-05-03T08:06:10.780+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:13.301+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:13.305+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 44: 4400/20353
[2025-05-03T08:06:13.308+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:15.774+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:15.786+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 45: 4500/20353
[2025-05-03T08:06:15.788+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:16.530+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:16.532+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 46: 4600/20353
[2025-05-03T08:06:16.539+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:18.276+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:18.288+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 47: 4700/20353
[2025-05-03T08:06:18.291+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:21.173+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:21.176+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 48: 4800/20353
[2025-05-03T08:06:21.178+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:23.316+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:23.316+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 49: 4900/20353
[2025-05-03T08:06:23.318+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:23.421+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:23.424+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 50: 5000/20353
[2025-05-03T08:06:23.425+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:24.270+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:24.272+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 51: 5100/20353
[2025-05-03T08:06:24.275+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:26.961+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:27.121+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 52: 5200/20353
[2025-05-03T08:06:27.339+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:30.371+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:30.409+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 53: 5300/20353
[2025-05-03T08:06:30.432+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:33.701+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:33.768+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 54: 5400/20353
[2025-05-03T08:06:33.909+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:35.635+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:35.636+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 55: 5500/20353
[2025-05-03T08:06:35.637+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:36.582+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:36.583+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 56: 5600/20353
[2025-05-03T08:06:36.585+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:37.278+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:37.300+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 57: 5700/20353
[2025-05-03T08:06:37.324+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:37.673+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:37.675+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 58: 5800/20353
[2025-05-03T08:06:37.678+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:38.917+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:38.937+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 59: 5900/20353
[2025-05-03T08:06:38.940+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:39.173+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:39.175+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 60: 6000/20353
[2025-05-03T08:06:39.182+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:40.156+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:40.170+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 61: 6100/20353
[2025-05-03T08:06:40.180+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:41.151+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:41.153+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 62: 6200/20353
[2025-05-03T08:06:41.160+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:41.352+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:41.352+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 63: 6300/20353
[2025-05-03T08:06:41.354+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:42.933+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:42.935+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 64: 6400/20353
[2025-05-03T08:06:42.939+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:43.430+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:43.431+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 65: 6500/20353
[2025-05-03T08:06:43.448+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:45.357+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:45.375+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 66: 6600/20353
[2025-05-03T08:06:45.410+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:45.789+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:45.791+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 67: 6700/20353
[2025-05-03T08:06:45.795+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:46.274+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:46.280+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 68: 6800/20353
[2025-05-03T08:06:46.281+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:47.611+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:47.634+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 69: 6900/20353
[2025-05-03T08:06:47.686+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:48.752+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:48.770+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 70: 7000/20353
[2025-05-03T08:06:48.777+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:49.097+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:49.099+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 71: 7100/20353
[2025-05-03T08:06:49.102+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:49.915+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:49.919+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 72: 7200/20353
[2025-05-03T08:06:49.965+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:51.832+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:51.836+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 73: 7300/20353
[2025-05-03T08:06:51.838+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:53.204+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:53.205+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 74: 7400/20353
[2025-05-03T08:06:53.211+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:53.899+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:53.903+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 75: 7500/20353
[2025-05-03T08:06:53.910+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:54.850+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:54.855+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 76: 7600/20353
[2025-05-03T08:06:54.893+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:55.798+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:55.842+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 77: 7700/20353
[2025-05-03T08:06:55.891+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:06:57.348+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:06:57.396+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 78: 7800/20353
[2025-05-03T08:06:57.503+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:01.306+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:01.482+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 79: 7900/20353
[2025-05-03T08:07:01.641+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:02.748+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:02.790+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 80: 8000/20353
[2025-05-03T08:07:02.898+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:03.551+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:03.560+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 81: 8100/20353
[2025-05-03T08:07:03.576+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:05.073+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:05.092+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 82: 8200/20353
[2025-05-03T08:07:05.109+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:05.458+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:05.458+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 83: 8300/20353
[2025-05-03T08:07:05.468+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:06.517+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:06.531+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 84: 8400/20353
[2025-05-03T08:07:06.534+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:08.498+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:08.533+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 85: 8500/20353
[2025-05-03T08:07:08.537+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:10.452+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:10.511+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 86: 8600/20353
[2025-05-03T08:07:10.524+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:13.375+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:13.395+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 87: 8700/20353
[2025-05-03T08:07:13.405+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:17.469+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:17.473+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 88: 8800/20353
[2025-05-03T08:07:17.546+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:20.315+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:20.340+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 89: 8900/20353
[2025-05-03T08:07:20.370+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:22.492+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:22.506+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 90: 9000/20353
[2025-05-03T08:07:22.618+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:23.152+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:23.191+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 91: 9100/20353
[2025-05-03T08:07:23.339+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:27.378+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:27.595+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 92: 9200/20353
[2025-05-03T08:07:27.635+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:30.144+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:30.293+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 93: 9300/20353
[2025-05-03T08:07:30.383+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:34.917+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:34.954+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 94: 9400/20353
[2025-05-03T08:07:34.972+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:40.657+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:41.337+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 95: 9500/20353
[2025-05-03T08:07:41.672+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:52.660+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:52.855+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 96: 9600/20353
[2025-05-03T08:07:52.935+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:56.100+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:56.177+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 97: 9700/20353
[2025-05-03T08:07:56.295+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:07:58.591+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:07:58.599+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 98: 9800/20353
[2025-05-03T08:07:58.621+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:00.369+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:00.443+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 99: 9900/20353
[2025-05-03T08:08:00.487+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:04.224+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:05.028+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 100: 10000/20353
[2025-05-03T08:08:05.404+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:08.684+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:08.884+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 101: 10100/20353
[2025-05-03T08:08:09.157+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:13.119+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:13.258+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 102: 10200/20353
[2025-05-03T08:08:13.834+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:17.118+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:17.137+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 103: 10300/20353
[2025-05-03T08:08:17.176+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:17.741+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:17.742+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 104: 10400/20353
[2025-05-03T08:08:17.749+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:20.935+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:20.946+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 105: 10500/20353
[2025-05-03T08:08:20.952+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:22.880+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:23.108+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 106: 10600/20353
[2025-05-03T08:08:23.286+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:26.394+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:26.867+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 107: 10700/20353
[2025-05-03T08:08:27.077+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:30.885+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:30.886+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 108: 10800/20353
[2025-05-03T08:08:30.887+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:31.876+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:31.877+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 109: 10900/20353
[2025-05-03T08:08:31.925+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:34.553+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:34.655+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 110: 11000/20353
[2025-05-03T08:08:34.666+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:35.314+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:35.325+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 111: 11100/20353
[2025-05-03T08:08:35.336+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:37.403+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:37.467+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 112: 11200/20353
[2025-05-03T08:08:37.490+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:08:47.052+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:08:47.190+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 113: 11300/20353
[2025-05-03T08:08:47.984+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:00.744+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:00.840+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 114: 11400/20353
[2025-05-03T08:09:00.881+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:03.278+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:03.359+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 115: 11500/20353
[2025-05-03T08:09:03.754+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:09.043+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:09.149+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 116: 11600/20353
[2025-05-03T08:09:09.260+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:11.192+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:11.193+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 117: 11700/20353
[2025-05-03T08:09:11.208+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:12.308+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:12.310+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 118: 11800/20353
[2025-05-03T08:09:12.323+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:12.588+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:12.600+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 119: 11900/20353
[2025-05-03T08:09:12.604+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:12.873+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:12.874+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 120: 12000/20353
[2025-05-03T08:09:12.965+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:13.665+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:13.695+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 121: 12100/20353
[2025-05-03T08:09:13.697+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:16.481+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:16.486+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 122: 12200/20353
[2025-05-03T08:09:16.492+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:19.722+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:19.953+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 123: 12300/20353
[2025-05-03T08:09:20.231+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:23.071+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:23.146+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 124: 12400/20353
[2025-05-03T08:09:23.180+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:25.924+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:25.933+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 125: 12500/20353
[2025-05-03T08:09:25.935+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:32.783+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:32.866+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 126: 12600/20353
[2025-05-03T08:09:32.890+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:37.227+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:37.635+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 127: 12700/20353
[2025-05-03T08:09:37.786+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:40.384+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:40.388+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 128: 12800/20353
[2025-05-03T08:09:40.393+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:44.516+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:44.679+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 129: 12900/20353
[2025-05-03T08:09:44.964+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:47.421+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:47.468+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 130: 13000/20353
[2025-05-03T08:09:47.474+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:50.603+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:50.607+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 131: 13100/20353
[2025-05-03T08:09:50.610+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:09:56.263+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:09:56.969+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 132: 13200/20353
[2025-05-03T08:09:57.439+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:10:06.490+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:10:06.534+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 133: 13300/20353
[2025-05-03T08:10:06.581+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:10:11.432+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:10:11.433+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 134: 13400/20353
[2025-05-03T08:10:11.530+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:10:14.355+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:10:14.373+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 135: 13500/20353
[2025-05-03T08:10:14.448+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:10:20.878+0000] {sql.py:580} INFO - Done loading. Loaded a total of 100 rows into diabetes_validation_processed
[2025-05-03T08:10:21.108+0000] {process_and_clean_data.py:745} INFO -   - Insertado lote 136: 13600/20353
[2025-05-03T08:10:21.140+0000] {base.py:84} INFO - Using connection ID 'mysql_diabetes_conn' for task execution.
[2025-05-03T08:14:05.809+0000] {job.py:218} ERROR - Job heartbeat got an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 691, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.8/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/MySQLdb/connections.py", line 195, in __init__
    super().__init__(*args, **kwargs2)
MySQLdb.OperationalError: (2005, "Unknown server host 'mysql' (-3)")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/jobs/job.py", line 192, in heartbeat
    self._merge_from(Job._fetch_from_db(self, session))
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/api_internal/internal_api_call.py", line 115, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/retries.py", line 89, in wrapped_function
    for attempt in run_with_db_retries(max_retries=retries, logger=logger, **retry_kwargs):
  File "/home/airflow/.local/lib/python3.8/site-packages/tenacity/__init__.py", line 347, in __iter__
    do = self.iter(retry_state=retry_state)
  File "/home/airflow/.local/lib/python3.8/site-packages/tenacity/__init__.py", line 325, in iter
    raise retry_exc.reraise()
  File "/home/airflow/.local/lib/python3.8/site-packages/tenacity/__init__.py", line 158, in reraise
    raise self.last_attempt.result()
  File "/usr/local/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/local/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/retries.py", line 98, in wrapped_function
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/jobs/job.py", line 316, in _fetch_from_db
    session.merge(job)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 3056, in merge
    return self._merge(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 3136, in _merge
    merged = self.get(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 2853, in get
    return self._get_impl(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 2975, in _get_impl
    return db_load_fn(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/loading.py", line 530, in load_on_pk_identity
    session.execute(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 1716, in execute
    conn = self._connection_for_bind(bind)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 1555, in _connection_for_bind
    return self._transaction._connection_for_bind(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/orm/session.py", line 750, in _connection_for_bind
    conn = bind.connect()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/future/engine.py", line 412, in connect
    return super(Engine, self).connect()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3325, in connect
    return self._connection_cls(self, close_with_result=close_with_result)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
    else engine.raw_connection()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3404, in raw_connection
    return self._wrap_pool_connect(self.pool.connect, _connection)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3374, in _wrap_pool_connect
    Connection._handle_dbapi_exception_noconnection(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 2208, in _handle_dbapi_exception_noconnection
    util.raise_(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3371, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 327, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 894, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 493, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 273, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 388, in __init__
    self.__connect()
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 691, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 211, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 686, in __connect
    self.dbapi_connection = connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 574, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 598, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.8/site-packages/MySQLdb/__init__.py", line 121, in Connect
    return Connection(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/MySQLdb/connections.py", line 195, in __init__
    super().__init__(*args, **kwargs2)
sqlalchemy.exc.OperationalError: (MySQLdb.OperationalError) (2005, "Unknown server host 'mysql' (-3)")
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2025-05-03T08:14:34.032+0000] {job.py:226} ERROR - Job heartbeat failed with error. Scheduler is in unhealthy state
